### ISSUE TITLE
V-Ray Renderer does not recognize Relative Paths

### ISSUE THREAD

**Issue by galagast** [ChaosGroup/vray-for-houdini#260]
1. In the Objects tab of V-Ray, if I use Absolute Paths like these:
![image](https://user-images.githubusercontent.com/16534677/32120091-eac69fe4-bb89-11e7-9be3-a61379d8c482.png)
It detects the objects and renders fine:
![image](https://user-images.githubusercontent.com/16534677/32120103-f2fc0f3c-bb89-11e7-8360-bda9d23ca334.png)

2. But if I use Relative Paths (in this case, for the box):
![image](https://user-images.githubusercontent.com/16534677/32120264-8906a7bc-bb8a-11e7-99a6-ab7c9e9e3c35.png)
It's unable to detect the box:
![image](https://user-images.githubusercontent.com/16534677/32120152-21c2bf96-bb8a-11e7-93fa-958540fbc8d8.png)

3. For comparison, I used the same paths for Mantra (mix of absolute and relative paths). It works:
![image](https://user-images.githubusercontent.com/16534677/32120215-50862962-bb8a-11e7-87ef-02d4e42039dd.png)

* If I also use a relative paths for the camera, sometimes I get errors like` Camera not set`. But unfortunately I could not reproduce a repo step as of now.

Scene File: [rel_paths.zip](https://github.com/ChaosGroup/vray-for-houdini/files/1423109/rel_paths.zip)
vfh 1774 35047d3 hfs16.0.736

**Comment by dominik2608** [comment]
I have a similar problem with channel expressions in the images tab:
![channelexpression](https://user-images.githubusercontent.com/18614866/32126223-d5d03308-bb6f-11e7-94e0-39ab21f8df9f.PNG)
Getting this error (but it reads the path correctly I think):
[2017/Oct/27:23:36:10] VFH [Error] V-Ray: Cannot create output image file "E:/CGI/only_tests/vray_tests/render/001/glass.exr": No such file or directory (2)

vfh 1774 35047d3 hfs16.0.736

Cheers
Dominik

**Comment by bdancer** [comment]
@dominik2608  your issue is a bit different (full nested directory path should be created), could you pls report it separately?

**Comment by bdancer** [comment]
@dominik2608 yours should be fixed with https://github.com/ChaosGroup/vray-for-houdini/commit/d1a64b8555da8692b3335ffd519cadfa022db7bc.

**Comment by bdancer** [comment]
@galagast should be fixed with https://github.com/ChaosGroup/vray-for-houdini/commit/3ab87a203a95e68c58677edb070681be8c873c33.

**Comment by dominik2608** [comment]
@bdancer Cool thanks - I would skip opening a new issue then, right? ;-)

**Comment by galagast** [comment]
Cool, relative paths now work on Objects!

After testing these, I also tried the Camera.
It doesn't work if I set the camera path to be relative.
![image](https://user-images.githubusercontent.com/16534677/32274399-9d8d5572-bf41-11e7-93fb-d1c6a81f316f.png)

I'm getting the error I mentioned on the first post:
![image](https://user-images.githubusercontent.com/16534677/32274387-916c0662-bf41-11e7-8989-ffe54b94a247.png)

Here's the same sample scene, updated for cameras:
H16.0.736 Indie - [rel_paths_cam.zip](https://github.com/ChaosGroup/vray-for-houdini/files/1434064/rel_paths_cam.zip)
vfh 1809 3ab87a2 hfs16.0.736 2017-11-01
